lib/dependencies.sh

```diff
@@ -59,12 +59,28 @@ log_build_scripts() {
   fi
 }
 
+yarn_supports_frozen_lockfile() {
+  local yarn_version="$(yarn --version)"
+  # Yarn versions lower than 0.19 will crash if passed --frozen-lockfile
+  if [[ "$yarn_version" =~ ^0\.(16|17|18).*$ ]]; then
```

Comment:
List of all the yarn versions we support: https://nodebin.herokai.com/v1/yarn/linux-x64.txt